extensions/fairifier-rdf-extension/dtls/fairifier/Resource.java

```diff
@@ -1,27 +1,32 @@
+
 package org.dtls.fairifier;
 
-import java.io.StringWriter;
+import java.io.IOException;
+
+import org.apache.http.HttpException;
 
 /**
  * @author Shamanou van Leeuwen
  * @date 28-11-2016
  *
  */
 public abstract class Resource {
+
     public String fairData = null;
-    
-    public void setFairData(String fairData){
+
+    public void setFairData(String fairData) {
         this.fairData = fairData;
     }
-    
-    protected boolean hasModel(){
+
+    protected boolean hasModel() {
         if (this.fairData == null) return false;
```

Comment:
[Missing curly brackets for an inline-if.](https://google.github.io/styleguide/javaguide.html#s4.1.1-braces-always-used)